frontend/src/components/RepositoryForm.vue

```diff
@@ -66,7 +68,7 @@
     </div>
     <div>
       <div class="textlabel">
-        {{ $t("common.branch") }} <span class="text-red-600">*</span>
+        {{ $t("common.branch") }}
```

Comment:
Branch in not required anymore?

Reply:
All fields are required except database group, but this was the only field marked star for required. So I'm removing it for not giving ambiguous guidance.